scylla-cdc-driver3/src/main/java/com/scylladb/cdc/cql/driver3/Driver3Cell.java

```diff
@@ -30,4 +29,26 @@ public Object getAsObject() {
     public DataType getDataType() {
         return columnDefinition.getCdcLogDataType();
     }
+
+    @SuppressWarnings("unchecked")
+    @Override
+    public Set<Field> getDeletedElements() {
+        if (!hasDeletedElements()) {
+            return null;
```

Comment:
Would it make sense to just return empty set here?

Reply:
Depends on how close to the CDC log you want to keep it. This is returning null, because the column in the log is null in this case.